### ISSUE TITLE
Release notes documentation does not include a date

### ISSUE THREAD

**Issue by jchesterpivotal** [concourse/concourse#248]
When I look at the [Release Notes page](http://concourse.ci/release-notes.html), I can see a list of release versions and their notes in descending order.

But I don't know when a particular release was made, and thus get a sense of how far behind I am in calendar terms. I also can't do simple sanity checking of my own dates in logs and the release note dates.

A person who guesses at the release workflow for Concourse might think to look at the [Github releases page](https://github.com/concourse/concourse/releases) for this information. But they might not think to do so. Even if they do, Github's interface is a cruel bugger that forces me to perform calendrical subtraction in order to get an actual date. Unnnnnless I happen to open it in an inspection view to look at the hidden data.

**tl;dr** "when the heck was v.xx.y released?" is too hard.

**Comment by vito** [comment]
does now
